cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest

```diff
@@ -28,7 +28,13 @@
   {% endif -%}
 {% endif -%}
 
-{% set params = "--master=127.0.0.1:8080" + " " + cluster_name + " " + cluster_cidr + " " + allocate_node_cidrs + " " + cloud_provider  + " " + cloud_config + service_account_key + pillar['log_level'] -%}
+{% set root_ca_file = "" -%}
+
+{% if grains['cloud'] is defined and grains.cloud in [ 'aws', 'gce' ]    %}
```

Comment:
FYI, the linked example just enables client cert auth to the API server selectively on aws/gce... it's not a statement of whether the CA cert exists only for those platforms.

I'm almost positive if server.{key,crt} exists that the ca.crt does as well... @roberthbailey do you know?